target/linux/mvebu/wdmc/base-files/lib/preinit/05_wdmc_config

```diff
@@ -0,0 +1,21 @@
+preinit_mount_config() {
+
+	. /lib/functions.sh
+
+	case $(board_name) in
+	armada-385-wd-mcex2u)
+		damaged=0
+		syscfg_part=$(grep config /proc/mtd |cut -c4)
+		ubiattach -m $syscfg_part -d 1 2>/dev/null || damaged=1
+		mkdir /tmp/wd-config
```

Comment:
you're reinventing the wheel here, just use (or extend) target/linux/mvebu/base-files/lib/preinit/81_linksys_syscfg

Reply:
It's almost just a copy of 81_linksys_syscfg, I did it only because this partition is needed by 06_set_iface_mac, 81 is way too late.

Reply:
then perhaps your `06_set_iface_mac` is wrong and you should do there something like `mac=$(mtd_get_mac_ascii config mac_addr)` instead of this weird `/tmp/wd-config` fiddling?

Reply:
The closest thing is mtd_get_mac_binary_ubi. Which is not really suitable, because there are evidences of the whole partition content being rewritten under some conditions in the original firmware (not all parts are open), and even in uboot they mount the partition to get the MAC. Reading it from a constant offset seems to be unreliable.

Reply:
Dead end (see below)